src/MainWindow.vala

```diff
@@ -215,23 +224,14 @@ namespace Terminal {
             search_menuitem.set_action_name (ACTION_PREFIX + ACTION_SEARCH);
             search_menuitem.add (new Granite.AccelLabel.from_action_name (_("Find…"), search_menuitem.action_name));
 
-            var show_in_file_browser_menuitem = new Gtk.MenuItem ();
-            show_in_file_browser_menuitem.set_action_name (ACTION_PREFIX + ACTION_OPEN_IN_FILES);
-            show_in_file_browser_menuitem.add (
-                new Granite.AccelLabel.from_action_name (
-                    _("Show in File Browser"),
-                    show_in_file_browser_menuitem.action_name
-                )
-            );
-
             menu = new Gtk.Menu ();
+            menu.append (open_in_browser_menuitem);
```

Comment:
If this is moved (Which is fine, this is probably more consistent with menuitem placement elsewhere), we probably need to add a separator beneath it

Reply:
OK, done.  It was moved to match the position of the "Open In" item in Files.